### ISSUE TITLE
Add an example of "how to make a file public" in the docs

### ISSUE THREAD

**Issue by jgeewax** [GoogleCloudPlatform/gcloud-node#350]
I've heard from a lot of people that this is the most common request, so it'd be really cool if we had an example in our docs.

ie:

---

To make an object publicly readable, you might do:

```nodejs
var storage = gcloud.storage();
var albums = storage.bucket('albums');

albums.acl.add({
  scope: 'allUsers',
  permission: Storage.acl.READER_ROLE
}, function(err, aclObject) {});
```

**Comment by ryanseys** [comment]
Yes :+1: absolutely. This is implemented as a simple check-box in the google developer console. What about even a dedicated function for this such as:

``` js
albums.acl.makePublic(function() {}); // set to allUsers reader
albums.acl.makePrivate(function() {}); // set to no permissions (default)
```

**Comment by jgeewax** [comment]
Somewhat dependent on what we expose per #351

**Comment by stephenplusplus** [comment]
Integrating `makePublic` and `makePrivate` over in #351. Merged #357 to address the lacking docs issue.
